rust/rope/src/rope.rs

```diff
@@ -756,6 +756,16 @@ impl<'a> From<&'a Rope> for String {
     }
 }
 
+impl fmt::Display for Rope {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut ci = self.iter_chunks(..);
+        for s in ci.next() {
```

Comment:
my only other observation here is that in rust you would normally just write this as,

```rust
for s in self.iter_chunks(..) {
// do something
}
```

That is to say, you don't need the extra step of assigning the iterator to a variable, you can just use it directly.

Reply:
Great. Will make that change.